src/ast/passes/codegen_llvm.cpp

```diff
@@ -1088,6 +1088,31 @@ void CodegenLLVM::visit(Call &call)
     b_.CreateLifetimeEnd(buf);
     expr_ = nullptr;
   }
+  else if (call.func == "bswap")
+  {
+    bpftrace::ast::Expression *arg = call.vargs->at(0);
+    auto scoped_del_arg = accept(arg);
+
+    assert(arg->type.IsIntegerTy());
+    if (arg->type.GetSize() > 1)
```

Comment:
ah yeah skipping the byte case makes sense.

re assert, we should try to avoid them as the end user experience of asserts is quite bad. But in this case have test coverage for it (`test("kprobe:f { bswap(\"hello\"); }", 10);`) so we will probably catch it before it ends up in the master branch